hut_10sqft/config/emacs/emacs_130s-serval.el

```diff
@@ -7,8 +7,32 @@
 ; 20170818 Comment out since this causes server error upon launch
 ;;(server-start)
 
+; 2/17/2012/To enabele run-lisp
```

Comment:
This block may be moved to somewhere upstream than a host-specific file.

Reply:
For now decided these are unimportant and would affect unintentionally the other Ubuntu hosts if moved to upstream so kept here.